Sources/TuistGraph/Models/Target.swift

```diff
@@ -31,8 +31,11 @@ public struct Target: Equatable, Hashable, Comparable, Codable {
     public var settings: Settings?
     public var dependencies: [TargetDependency]
     public var sources: [SourceFile]
+    public var sourcesPaths: [AbsolutePath]
     public var resources: ResourceFileElements
+    public var resourcesPaths: [AbsolutePath]
     public var copyFiles: [CopyFilesAction]
+    public var copyFilesPaths: [AbsolutePath]
```

Comment:
Do we actually need to track these paths? Can't we lint that the files added do exist based on directly the `sources`, `resources`, and `copyFiles` properties?

Reply:
Well, i tried to make it work but couldnt since we kinda need to find a point of reference. For example, `/app/sources/text.swift` will not show up in the `sources` if its not within the directory. So we kinda need to store the paths to compare them later on in the linter.

Reply:
There should be a place where we skip adding files to `sources` when they don't exist. Maybe we can try including them? Would be interesting to see what happens as that should be a valid operation in Xcode.